### PR TITLE
UI changes intake search filter form

### PR DESCRIPTION
**Pull request by louh**
## What does this change?

This re-arranges intake search form filter UI as described by https://github.com/usdoj-crt/crt-portal-management/issues/1011.

While positioning and functionality is done, it's the details that might need some tweaking. Please see the discussion in that issue for more details.

## Screenshots (for front-end PR):

image

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [x] Check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.
+ [ ] Re-check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

**Comment by ilodidoj** [comment]
@louh I changed the PR title and removed pro-form reference. Pro-Form is the "Add new record" form not the search filter form.

**Comment by billyhunt** [comment]
I notice that the asignee field goes off screen with the new design on mobile.  Not sure if we support that, but its something to consider.  
image

**Comment by billyhunt** [comment]
Actually, nevermind.  We don't support mobile for the internal tool, as you can only access it from a DOJ computer.

**Review by billyhunt — APPROVED** [review]
Looks good to me.

**Review by ilodidoj — APPROVED** [review]
UI working as expected.

File: crt_portal/static/sass/custom/intake.scss [hunk 1]
```diff
@@ -193,24 +197,11 @@
     }
 
     .usa-button {
+      margin-right: 1em;
```
[the remark this answers is not in the document]
Comment (louh) [reply crt_portal/static/sass/custom/intake.scss:200]: Yep - it's pretty narrow. It'll only apply to children of `.intake-content .intake-filters` which only appear on this part of the site.

**Pull request merged**